qiskit/ignis/verification/randomized_benchmarking/fitters.py

```diff
@@ -400,10 +400,9 @@ def fit_data(self):
         for patt_ind, _ in enumerate(self._rb_pattern):
 
             qubits = self._rb_pattern[patt_ind]
-            fit_guess = [0.95, 0.99, 0.0]
-            fit_guess[0] = self._ydata[patt_ind]['mean'][0]
```

Comment:
This was also a little incorrect, this should have been `self._ydata[patt_ind]['mean'][0]-1/2**len(qubits)`

Reply:
This is y0 - fit_guess[2] that appears later. This PR adds the condition that y0 is larger.